tests/src/test/scala/slinky/core/TagTest.scala

```diff
@@ -28,7 +27,7 @@ class TagTest extends FunSuite {
   }
 
   test("Sequence of different tag types can be typed to TagComponent[Any]") {
-    val foo: Seq[ReactElement] = Seq(div(), a())
+    Seq(div(), a())
```

Comment:
This still needs to be converted to a `Seq[ReactElement]` to test the implicit conversions.